team13-fydp/fydp
# Excel Sheet Update

Added in FTE check, error validation for classes and teachers.

To test this branch please do the following:
- open eclipse, configure POIs, ensure there are no errors in the code
- ensure that the file path is pointing to the new file (feb-4-front-end.xlsx)
- go to where your repo is (MAMP -> htdocs -> fydp) and ensure the file is found there (feb-4-front-end.xlsx)
- test the file in the following ways:
- add x's to all available slots for a specific teacher (all morning and afternoon cells) and ensure that an 'x' populates in the Full Time column and a 1 in the FTE column
- go to input sheet 2 and fill out step 4 (# of teachers): put an integer that **matches and doesn't match ** in the # of teachers column and see that the error validation prompts you to fix it
- fill out # of french teachers that **matches and doesn't match** the french certified teachers column on the sheet before
- go to input sheet 3 and fill out the # of classes at the school with an integer that **matches and doesn't match** the number of classes on the previous page, check the error validation
- try clicking and editing the instructions on all 3 input sheets, make sure an error is received
- try clicking and adding in the proper cells (i.e. add a new teachers FTE on input sheet 1, a homeroom teacher on input sheet 2 and specialty teacher on input sheet 3) and make sure it lets you edit it
